packages/ui/src/components/va-menu/VaMenu.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <va-dropdown class="va-dropdown" v-model="doShowDropdown" v-bind="dropdownProps">
+    <template #anchor>
+      <slot name="anchor">
+        <VaButton>
+          Слава Україні!!!
+        </VaButton>
+      </slot>
+    </template>
+    <template #default>
+      <VaDropdownContent :no-padding="true">
+        <va-menu-list :options="options" @option-click="$emit('option-click', $event)">
```

Comment:
We need to cover this component with slots:

- option - replace whole option
- left-icon - replace left icon (for example with Avatar)
- right-icon - replace right icon
- option-text - replace text
- options - replace whole options list v-for, so user can render VaMenuItem manually

```html
<VaMenu>
   <template #options>
       <VaMenuItem>My profile</VaMenuItem>
       <VaMenuItem>Settings</VaMenuItem>
       <VaDivider />
       <VaMenuItem style="color: red">Log out</VaMenuItem>
   </template>
</VaMenu>
```